torch/csrc/utils/tensor_new.cpp

```diff
@@ -208,6 +208,7 @@ Tensor internal_new_from_data(
     bool type_inference,
     bool pin_memory = false) {
 
+
```

Comment:
avoid adding random whitespace changes in PRs

Reply:
Will keep that in mind. Thank you.